projects/angular-cesium/src/lib/angular-cesium-widgets/models/editable-rectangle.ts

```diff
@@ -141,10 +141,13 @@ export class EditableRectangle extends AcEntity {
       this.lastDraggedToPosition = startMovingPosition;
     }
 
-    const delta = GeoUtilsService.getPositionsDelta(this.lastDraggedToPosition, draggedToPosition);
+    const lastDraggedCartographic = Cesium.Cartographic.fromCartesian(this.lastDraggedToPosition);
+    const draggedToPositionCartographic = Cesium.Cartographic.fromCartesian(draggedToPosition);
     this.getRealPoints().forEach(point => {
-      const newPos = GeoUtilsService.addDeltaToPosition(point.getPosition(), delta);
-      point.setPosition(newPos);
+      const cartographic = Cesium.Cartographic.fromCartesian(point.getPosition());
```

Comment:
Converting the drag positions to cartographics and then altering the points based on the delta calculated from them means that, in 2d mode, the rectangle's width doesn't change now and it looks good. In 3d mode, dragging left to right and vice-versa doesn't alter the height of the rectangle now. However, when you drag the rectangle near the poles, the rectangle begins to wrap around them.

Reply:
I *think* this is due to Rectangles being defined with radians.

Reply:
Yeah, it looks like this is expected behavior with RectangleGraphics -> 

> Describes graphics for a Rectangle. The rectangle conforms to the curvature of the globe and can be placed on the surface or at altitude and can optionally be extruded into a volume.